common/pupil/registration.ts

```diff
@@ -90,6 +90,7 @@ export async function registerPupil(data: RegisterPupilData, noEmail = false, pr
             isPupil: true,
 
             // the authToken is used to verify the e-mail instead
+            // PR Note: How do we drop this?
             verification,
```

Comment:
No need to drop this for now, we can do it in a separate PR (the same that drops the column from the `schema.prisma`)

Reply:
Could you remove the comment or change it to `TODO: Remove after migration`?
You can leave PR Notes as a comments here in Github.